glossary/flyscanning.md

```diff
@@ -0,0 +1,23 @@
+# Flyscanning
```

Comment:
Hmm, this is a definition like all the others but is also the page title. I'd like the flyscanning definition to be consistent with the others but the following seems pretty redundant:

```suggestion
# Flyscanning

## Flyscanning
```

Any thoughts?

Reply:
Ok I'll, change that